umap/static/umap/js/umap.js

```diff
@@ -240,6 +241,42 @@ U.Map = L.Map.extend({
     this.on('click contextmenu.show', this.closeInplaceToolbar)
   },
 
+  render: function (fields) {
+    let impacts = U.Utils.getImpactsFromSchema(fields)
```

Comment:
If the schema is static, don't we want to compute that once and for all? (vs. at each `render()` call)

Reply:
The schema is static but the list of passed fields isn't.`getImpactsFromSchema` goes trough the keys of the schema, returning what is impacted by the given fields.

I don't think the lookup will take much time. If it is, we might want to cache out the results, maybe. I'd say it's an optimisation for the future, when/if we need it. Would that be okay for you?

Reply:
Alright 👍